modules/apps/josh-lapurd-theme/src/WEB-INF/src/resources-importer/journal/structures/text_Paragraph_With_Header.json

```diff
@@ -0,0 +1,112 @@
+{
+    "availableLanguageIds": [
+        "en_US"
+    ],
+    "defaultLanguageId": "en_US",
+    "fields": [
+        {
+            "label": {
+                "en_US": "Select"
+            },
+            "options": [
+                {
+                    "value": "",
+                    "label": {
+                        "en_US": "none"
+                    }
+                },
+                {
+                    "value": "dark",
+                    "label": {
+                        "en_US": "dark"
+                    }
+                },
+                {
+                    "value": "light",
+                    "label": {
+                        "en_US": "light"
+                    }
+                },
+                {
+                    "value": "light-2",
+                    "label": {
+                        "en_US": "light-2"
+                    }
+                },
+                {
+                    "value": "white",
+                    "label": {
+                        "en_US": "white"
+                    }
+                }
+            ],
+            "predefinedValue": {
+                "en_US": [
+                    ""
+                ]
+            },
+            "style": {
+                "en_US": ""
+            },
+            "tip": {
+                "en_US": "Select background color for this module."
+            },
+            "dataType": "string",
+            "indexType": "keyword",
+            "localizable": true,
+            "multiple": false,
+            "name": "bg_Color",
```

Comment:
Rename to ```bg_color```.  The text does not need to be capitalized because it is spaced by an underscore.